### PR TITLE
Improve code comments

### LINKED ISSUE #644: Remove PVS-Studio comments
A number of files start with a comment:
```cpp
// This is an open source non-commercial project. Dear PVS-Studio, please check it.
// PVS-Studio Static Code Analyzer for C, C++ and C#: http://www.viva64.com
```

These comments appear to be obsolete now, and should be removed.

----

PVS-Studio was a local plugin tool that would scan the source code for errors. It required that comment to be present to allow for free scanning. As Visual Studio has built in scanning, which is apparently as good or better, the plugin is no longer being used.

### PR DESCRIPTION
**Pull request by Brett208**
Closes #644 
Partially address #676
Partially address #677

### CODE REVIEW

**Comment by Brett208** [comment]
I know the issues this PR addresses are marked for V1.0.0, but I thought it would be good to generally clean the code now. That way people won't use the older style as an example within the code base.

File: OPHD/UI/Core/Slider.cpp [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * \file	Slider.c++
+ * \file	Slider.cpp
  * \brief	Functions for sliding controls.
  * \author	Goof
```
Comment (DanRStevens) [inline OPHD/UI/Core/Slider.cpp:4]: Bit of a side note, though I've read that when author's are listed in the source file of open source projects, it tends to discourage other people form maintaining the file. Perhaps we should omit author comments from source files. The information is still part of the commit meta data. Similar for comments that simply state the date a change was made.

**Review by DanRStevens — APPROVED** [review]
Thank you for taking care of that.

File: OPHD/UI/Core/Slider.cpp (hunk 1 above)
Comment (DanRStevens) [reply OPHD/UI/Core/Slider.cpp:4]: ... of course, I've never felt comfortable removing someone else's author comments. But then, that's maybe getting to the heart of the above.

**Comment by ldicker83** [comment]
It's marked for 1.0.0 but that doesn't mean it needs to wait for that. For me it's a way of prioritizing tasks mostly for myself and just getting them assigned so that by 1.0.0 all of the tasks are completed. I have no objections to taking on tasks that are assigned for other milestones earlier especially if they're not related to core gameplay. :D

File: OPHD/UI/Core/Slider.cpp (hunk 1 above)
Comment (Brett208) [reply OPHD/UI/Core/Slider.cpp:4]: I had mixed thoughts about this as well. I think you are right about removing attribution unless it is a large module like the Pathfinder or something.

Comment (ldicker83) [reply OPHD/UI/Core/Slider.cpp:4]: Author notes are an artifact of the past when source control wasn't a thing. Author information is in the repository commit history, I see very low value in keeping authorship comments in place. Including the file name also makes no real sense.

**Pull request merged**